[2021-04-27 17:57:35,937] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.print_date 2021-04-25T02:20:00+00:00 [queued]>
[2021-04-27 17:57:35,965] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.print_date 2021-04-25T02:20:00+00:00 [queued]>
[2021-04-27 17:57:35,966] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-27 17:57:35,967] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-04-27 17:57:35,968] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-27 17:57:35,985] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): print_date> on 2021-04-25T02:20:00+00:00
[2021-04-27 17:57:35,990] {standard_task_runner.py:52} INFO - Started process 117 to run task
[2021-04-27 17:57:35,996] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'docker_sample', 'print_date', '2021-04-25T02:20:00+00:00', '--job-id', '29', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dockerOperator.py', '--cfg-path', '/tmp/tmp25mjlyqg', '--error-file', '/tmp/tmpizaxeqrq']
[2021-04-27 17:57:35,997] {standard_task_runner.py:77} INFO - Job 29: Subtask print_date
[2021-04-27 17:57:36,068] {logging_mixin.py:104} INFO - Running <TaskInstance: docker_sample.print_date 2021-04-25T02:20:00+00:00 [running]> on host 44e91a58ae1b
[2021-04-27 17:57:36,125] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=docker_sample
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2021-04-25T02:20:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-25T02:20:00+00:00
[2021-04-27 17:57:36,126] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-04-27 17:57:36,128] {bash.py:158} INFO - Running command: date
[2021-04-27 17:57:36,138] {bash.py:169} INFO - Output:
[2021-04-27 17:57:36,141] {bash.py:173} INFO - Tue Apr 27 17:57:36 UTC 2021
[2021-04-27 17:57:36,143] {bash.py:177} INFO - Command exited with return code 0
[2021-04-27 17:57:36,179] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=docker_sample, task_id=print_date, execution_date=20210425T022000, start_date=20210427T175735, end_date=20210427T175736
[2021-04-27 17:57:36,218] {taskinstance.py:1220} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2021-04-27 17:57:36,253] {local_task_job.py:146} INFO - Task exited with return code 0
